config/crd/bases/core.openfeature.dev_featureflagconfigurations.yaml

```diff
@@ -240,6 +240,19 @@ spec:
                 properties:
                   name:
                     type: string
+                  remoteSyncConfiguration:
```

Comment:
Is http better than "remote"?  "Remote" seems a bit nonspecific.

Reply:
will update this to `http`